.github/workflows/ci.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        ruby: ['2.5', '2.6', '2.7', '3.0', '3.1']
+        ruby: ['2.6', '2.7', '3.0', '3.1']
```

Comment:
[Steep](https://github.com/soutaro/steep/) requires at least 2.6 version of ruby.
So we would like to have at least a 2.6 version of ruby to test with CI.
ref: https://github.com/soutaro/steep/blob/master/steep.gemspec#L29

---

Should this be handled in a separate PR?

Reply:
I have no information to decide it. Is there any reason to support Ruby 2.5? If so, we may need to support it 😮‍💨

Reply:
ruby 2.5 has already EOL. so it could remove :+1:

Reply:
During the course of the PR, the required ruby version of steep was raised to 2.7, so it was modified accordingly https://github.com/moneyforward/jct/pull/16/commits/41394665141570ec428f3266309604f9e61bbc4e
see-also: https://github.com/soutaro/steep/pull/696